### PR TITLE
Rename files settings on the bottom left

### PR DESCRIPTION
**Pull request by PVince81**
Part of https://github.com/nextcloud/server/issues/32350

image

### CODE REVIEW

File: apps/files/templates/appnavigation.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		<div id="app-settings-header">
 			<button class="settings-button"
 					data-apps-slide-toggle="#app-settings-content">
-				<?php p($l->t('Settings')); ?>
+				<?php p($l->t('Files Settings')); ?>
```
Comment (jancborchardt) [inline apps/files/templates/appnavigation.php:34]:
```suggestion
				<?php p($l->t('Files settings')); ?>
```

**Review by jancborchardt — CHANGES_REQUESTED** [review]

**Review by jancborchardt — APPROVED** [review]

**Review by nimishavijay — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by Pytal** [comment]
CI failure seems related to this https://github.com/nextcloud/server/blob/6c42874f04477a9741089bd4763ad47ad4130b2f/tests/acceptance/features/bootstrap/SettingsMenuContext.php#L143

**Comment by PVince81** [comment]
I'll push a follow up PR

**Comment by PVince81** [comment]
@Pytal I checked the related code and it all looks related to the top right menu, not the bottom left. weird

**Comment by PVince81** [comment]
@Pytal this is the actual issue: ` Nextcloud test server could not be started (Exception)`

it's just a coincidence that the test scenario contains a reference to settings, and here it means the settings page from the top right menu

**Pull request merged**